src/js/modules/scrollToAnchor.js

```diff
@@ -0,0 +1,22 @@
+import { closeMenu } from './mobileMenu.js';
+
+const anchors = Array.from(document.querySelectorAll("a[href*='#']"));
+
+anchors.forEach((anchor) => {
+	anchor.addEventListener("click", (event) => {
+		event.preventDefault();
+		closeMobileMenu();
+		const blockID = anchor.getAttribute("href");
+		document.querySelector(`${blockID}`).scrollIntoView({
```

Comment:
getElementById workds faster, you have id in href

Reply:
Done

Reply:
`blockID` - is already string in variable, you don't need this `${blockID}`, just use `getElementById(blockID)`

Reply:
Done